app.rb

```diff
@@ -75,18 +64,20 @@ def create_person
     create_person_object(select_person)
```

Comment:
- [ ] In `create_person_object` method you are getting input from users and creating the book, This method does not follow `single responsibility principle`. Please make sure to separate these operations into different methods to achieve `single responsibility principle`.